### PR TITLE
Remove useless parameter at documentation 

### PR DESCRIPTION
**Pull request by pedrommone**

### CODE REVIEW

**Comment by doomsayer13** [comment]
Hi @pedrommone ,
Thank you for your PR, we'll check it ASAP.

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To run the `xcodebuild` command and only show these lines you can add the
 postfix: `| grep -i -e 'Signing Identity' -e 'Provisioning Profile' -e '` to
```
Comment (viktorbenei) [inline README.md]: Good catch @pedrommone ! Should be removed from here too ;)

**Comment by viktorbenei** [comment]
Go 1.10 related thread: https://github.com/bitrise-tools/codesigndoc/issues/60

**Comment by pedrommone** [comment]
@viktorbenei done.

**Comment by pedrommone** [comment]
@viktorbenei is there a way to make those build results public? When it fails, we cannot check them! :(

**Comment by viktorbenei** [comment]
@pedrommone not yet, but should be soon ;)

**Comment by viktorbenei** [comment]
Thanks for the update @pedrommone !

Actually I just re-read the README and tested the command, and I think the last `-e` param got trimmed for some reason, but there is one more.

So, to get the output mentioned in the README at "By running this command you'll see an output similar to:"

```
Signing Identity:     "iPhone Developer: Viktor Benei (F...7)"
Provisioning Profile: "BuildAnything"
                      (9.......-....-....-....-...........0)
```

you also have to include the pattern for the UDID line:

```
grep -i -e 'Signing Identity' -e 'Provisioning Profile' -e '                      ('
```

Otherwise the UDID won't be included.

**Comment by pedrommone** [comment]
@viktorbenei I'm sorry but I cannot find where is wrong. Can you show the exacly point?

**Comment by viktorbenei** [comment]
@pedrommone sorry I didn't mean your PR is wrong, just discovered that the last "empty" `-e` did have a purpose originally, but was never committed into the README it seems, for some reason it got trimmed/clipped in the process..

Instead of `-e '` it should have been `-e '                      ('`

Otherwise the UUID line which is included in the README example will not be included in the `grep` output

```
                      (9.......-....-....-....-...........0)
```

**Comment by godrei** [comment]
Closing as we did not receive a reply - @pedrommone please let us know if we can help with anything and we'll reopen this issue!

**Pull request closed without merging**